### PR TITLE
Make commoncrawl bucket configurable, change default

### LINKED ISSUE #316: Make common crawl s3 bucket configurable
Currently, the common crawl DAG's s3 bucket is hardcoded to a bucket that is not owned by the WordPress project or even Creative Commons.

We should make this string configurable and set the default value to `ov-commonsmapper`.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #316 by @AetherUnbound

## Description

This PR changes the commoncrawl bucket to be configurable via environment variable. I've also reserved a different bucket for us to use when we end up wanting to run this again.

## Testing Instructions

I do not know how to run this so I'm just basing this off of `just lint` and `just test`.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/commoncrawl/commoncrawl_scripts/commoncrawl_s3_syncer/SyncImageProviders.py [hunk 1]
```diff
@@ -17,13 +18,12 @@
 def getCrawlIndex(_param):
 
     if not _param:  # get the most recent index from common crawl
-        bucket = "commoncrawl"
         s3 = boto3.client("s3", config=Config(signature_version=UNSIGNED))
 
         # verify bucket
         contents = []
         prefix = "cc-index/collections/CC-MAIN-"
```
Comment (obulat) [inline openverse_catalog/dags/commoncrawl/commoncrawl_scripts/commoncrawl_s3_syncer/SyncImageProviders.py:25]: I have no idea what this prefix is for, but is it still correct?

**Review by obulat — APPROVED** [review]
LGTM!

File: openverse_catalog/dags/commoncrawl/commoncrawl_scripts/commoncrawl_s3_syncer/SyncImageProviders.py (hunk 1 above)
Comment (AetherUnbound) [reply openverse_catalog/dags/commoncrawl/commoncrawl_scripts/commoncrawl_s3_syncer/SyncImageProviders.py:25]: I'm really not sure :sweat_smile: I assume there will be a whole host of things we'll need to change once we actually try and get this going!

**Review by zackkrida — APPROVED** [review]
I think this is sufficient until we actually rebuild this pipeline. Nice work!

**Pull request merged**